portfolio/src/main/java/com/google/sps/servlets/CommentsDataServlet.java

```diff
@@ -81,4 +73,21 @@ private String convertToJsonUsingGson(ArrayList<String> comments) {
     return json;
   }
 
+  private void populateCommentsList(HttpServletRequest request){
+    comments.clear();
+
+    int maxComments = Integer.parseInt(request.getParameter("maxComments"));
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    
+    Query query = new Query("comment");
+    PreparedQuery results = datastore.prepare(query);
+  
+    for (Entity entity : results.asIterable(FetchOptions.Builder.withLimit(maxComments))) {
+
+      String comment = (String) entity.getProperty("text");
```

Comment:
I think we should perform type checks whenever casting

Reply:
I'm not sure what you mean by type checks? Should I be doing a try Catch? or a if statetment using instanceof?